server/belga/io/feed_parsers/belga_newsml_mixin.py

```diff
@@ -34,3 +34,17 @@ def _get_countries(self, country_code):
             for c in countries
             if c.get('qcode') == country_code.lower() and c.get('is_active')
         ]
+
+    def _get_keywords(self, data):
+        belga_keywords = get_resource_service('vocabularies').find_one(req=None, _id='belga-keywords').get('items', [])
+
+        _belga_keyword_list = [
+            {'name': c['name'], 'qcode': c['qcode'], 'translations': c['translations'], 'scheme': 'belga-keywords'}
```

Comment:
why not use the `get_items` method?

Reply:
my bad, I will update it with get_items.